fastlane/Fastfile

```diff
@@ -38,24 +38,31 @@ platform :ios do
     end
   end
 
-  desc "Runs all the tests"
-  lane :test do |options|
-    # Scan can't target two different platforms in devices
-    # Need to run scan multiple times
+  desc "Runs all the iOS tests"
+  lane :test_ios do |options|
+    devices = options[:devices]&.split(',')
+    devices ||= ["iPhone 8 (14.5)", "iPhone 12 (15.2)"]
```

Comment:
Awesome, this defines a default, right?

Reply:
Yup yup! It's not my favorite way of doing it but it works for now 🤷‍♂️

Reply:
I don't know any Ruby, but I figured 😅